tests/page-objects/apdex/tasks.page.js

```diff
@@ -0,0 +1,20 @@
+const Page = require('./page');
```

Comment:
Because it is easier to understand and track when anyone reads the code. Also, it is about structure and organization, which is what we have been trying to improve in `/test` folder.

Here are some of the files that are inside `page-object` folder and reference the other files using `@page-objects`:

- [death-report.page.js](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/enketo/death-report.page.js#L3)
- [contacts.wdio.page.js](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/contacts/contacts.wdio.page.js#L1-L8)
- [search.wdio.page.js](https://github.com/medic/cht-core/blob/master/tests/page-objects/default-mobile/search/search.wdio.page.js#L1)
- [common.wdio.page.js](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/common/common.wdio.page.js#L3)
- [target-aggregates.wdio.page.js](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/targets/target-aggregates.wdio.page.js#L1)

The config of `@page-objects` is in the file [`tests/jsconfig.json`](https://github.com/medic/cht-core/blob/master/tests/jsconfig.json#L8)